pkg/field/context.go

```diff
@@ -4,6 +4,13 @@ import "context"
 
 // context.Context support for fields
 
+const (
+	ActionsetNameKey = "ActionSet"
```

Comment:
Not sure we discussed this in slack. These constants should be defined outside the `field` package. Both the interface and the implementation for `field` and the replacement `log` and `errors` packages should not contain kanister specific references or dependencies.